language/move-prover/bytecode/src/stackless_control_flow_graph.rs

```diff
@@ -16,10 +17,10 @@ type Map<K, V> = BTreeMap<K, V>;
 type Set<V> = BTreeSet<V>;
 pub type BlockId = CodeOffset;
 
-#[derive(Debug)]
-struct Block {
-    successors: Vec<BlockId>,
-    content: BlockContent,
+#[derive(Clone, Debug)]
+pub struct Block {
```

Comment:
Can we make all those things `pub(crate)` to avoid exporting details out of this crate?